lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -54,6 +60,32 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.DetailsJSON} text
+   * @return {!Element}
+   */
+  _renderURL(text) {
+    const element = this._renderText(text);
+    element.classList.add('lh-text__url');
+    return element;
+  }
+
+  /**
+   * @param {!DetailsRenderer.ThumbnailDetails} value
+   * @return {!Element}
+   */
+  _renderThumbnail(value) {
+    if (/^image/.test(value.mimeType) === false) {
+      return this._dom.createElement('span');
+    }
+
+    const element = this._dom.createElement('img', 'lh-thumbnail');
+    element.src = value.url;
+    element.alt = 'Image preview';
```

Comment:
was matching our v1. :)  i changed it to empty string, which seems better, actually.